### PR TITLE
Run e2e tests on compiled binaries

### PR DESCRIPTION
**Pull request by mcasimir**
Run e2e tests on compiled binaries

### CODE REVIEW

File: packages/cli-repl/test/test-shell.ts [hunk 1]
```diff
@@ -8,22 +8,31 @@ import stripAnsi from 'strip-ansi';
 import assert from 'assert';
 
 const PROMPT_PATTERN = /^> /m;
-const ERROR_PATTERN = /Thrown:\n([^>]*)/m;
```
Comment (aherlihy) [inline packages/cli-repl/test/test-shell.ts:11]: Should this go in a separate PR since it's a separate issue?

Comment (mcasimir) [reply packages/cli-repl/test/test-shell.ts:11]: Without that tests break when running on the compiled binary since the node version is  slightly different from the one downloaded in the CI vm
